### PR TITLE
add in a data only container

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

**Comment by andymckay** [comment]
For https://github.com/mozilla/payments-env/pull/10

**Comment by andymckay** [comment]
For https://github.com/mozilla/payments-env/pull/10

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM mozillamarketplace/centos-python27-mkt:0.5
+
+COPY . /srv/payments-ui
```
Comment (muffinresearch) [inline Dockerfile]: Should be the contents of public not everything from the project root.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM mozillamarketplace/centos-python27-mkt:0.5
```
Comment (muffinresearch) [inline Dockerfile]: Iis there any advantage to building on top of the centos image as opposed to busybox? I guess all these layers (for centos/python) are there (cos we're using them elsewhere) - so size shouldn't be any issue.

File: Dockerfile (hunk 1 above)
Comment (andymckay) [inline Dockerfile]: Ah right.

File: Dockerfile (hunk 2 above)
Comment (andymckay) [inline Dockerfile]: As we chatted its cached. We've already got busybox which is tiny, may as well use that.

**Comment by muffinresearch** [comment]
lgtm r+

**Pull request merged**